clustered-indexes.md

```diff
@@ -0,0 +1,188 @@
+
+```sql
+EXPLAIN SELECT id FROM always_clusters_in_all_versions WHERE b = 'aaaa';
+EXPLAIN SELECT guid FROM does_not_cluster_by_default WHERE b = 'aaaa';
+```
+
+```sql
++--------------------------+---------+-----------+---------------------------------------------------+-------------------------------------------------------+
+| id                       | estRows | task      | access object                                     | operator info                                         |
++--------------------------+---------+-----------+---------------------------------------------------+-------------------------------------------------------+
+| Projection_4             | 0.00    | root      |                                                   | test.always_clusters_in_all_versions.id               |
+| └─IndexReader_6          | 0.00    | root      |                                                   | index:IndexRangeScan_5                                |
+|   └─IndexRangeScan_5     | 0.00    | cop[tikv] | table:always_clusters_in_all_versions, index:b(b) | range:["aaaa","aaaa"], keep order:false, stats:pseudo |
++--------------------------+---------+-----------+---------------------------------------------------+-------------------------------------------------------+
+3 rows in set (0.01 sec)
+
++---------------------------+---------+-----------+-----------------------------------+------------------------------------------------+
+| id                        | estRows | task      | access object                     | operator info                                  |
++---------------------------+---------+-----------+-----------------------------------+------------------------------------------------+
+| Projection_4              | 0.00    | root      |                                   | test.does_not_cluster_by_default.guid          |
+| └─TableReader_7           | 0.00    | root      |                                   | data:Selection_6                               |
+|   └─Selection_6           | 0.00    | cop[tikv] |                                   | eq(test.does_not_cluster_by_default.b, "aaaa") |
+|     └─TableFullScan_5     | 2.00    | cop[tikv] | table:does_not_cluster_by_default | keep order:false, stats:pseudo                 |
++---------------------------+---------+-----------+-----------------------------------+------------------------------------------------+
+4 rows in set (0.00 sec)
+```
+
+# With clustered indexes
+
+New to TiDB 5.0-rc, clustering is permitted by any `PRIMARY KEY`. The following `EXPLAIN` output shows the previous example with clustered indexes now enabled:
```

Comment:
```suggestion
Since v5.0, TiDB supports clustering by any `PRIMARY KEY`. The following `EXPLAIN` output shows the previous example with clustered indexes enabled:
```